packages/truffle-db/src/loaders/test/index.ts

```diff
@@ -3,6 +3,7 @@ import path from "path";
 import gql from "graphql-tag";
 import { TruffleDB } from "truffle-db";
 import * as Contracts from "truffle-workflow-compile";
+import * as Ganache from "ganache-core"
```

Comment:
I don't think this is used in this file?

Reply:
Oops, you're right! Will remove.